drivers/bridge/bridge.go

```diff
@@ -570,6 +570,10 @@ func (d *driver) getNetworks() []*bridgeNetwork {
 func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) error {
 	var err error
 
+	if d.config == nil {
+		return fmt.Errorf("bridge driver configuration is missing")
```

Comment:
In libentwork we moved away from static defined errors in favor of error interfaces (look at [error interfaces](https://github.com/docker/libnetwork/blob/master/types/types.go#L288)).

This works especially well for the rest server.
We did not want the client to parse error strings but neither we want to maintain a big list of statically defined error types because at this time it is hard to come out with a type which will not change in future and will always be valid.

Going back to this specific case, I should not return a generic error here (which web server would translate in an internal server error), rather a BadRequestError.

Thanks.
Will make the change.